### ISSUE TITLE
Input data contains at least one less than ideal video set

### ISSUE THREAD

**Issue by einsteinxx** [einsteinxx/UCLA_BIOENG_596_FALL_21#2]
Current trial training data may contain some images that are not wholly cropped within the US view and there's at least one "bad" (vertical striped light and dark patterns)  ultrasound image within my training or test set.

**Comment by einsteinxx** [comment]
Tensor image list needs to be reconstructed to pair up the image ID with the name (Faster RCNN format used didn't push names into data)

One of the test images with improper image selection:
![image](https://user-images.githubusercontent.com/27804848/182021195-51d0ca2a-5dc5-4628-9adc-20ea84c40a04.png)

  This one has the keystone shape instead of the rectangle like most of the others provided. I use the flat background of the GUI to delineate between US image and GUI, but this one looks to have been saved in a different view than the majority of the data.

  I get a FN on this one due to my classification score cutoff, but it did fixate on an area within the actual US area (tiny dark spot). I will leave this patient set in, for now. Writing some functions to make finding and pairing up image ids with patient test names.
